### PR TITLE
[17.0][MIG] stock_picking_volume_packaging

### PR DESCRIPTION
**Pull request by Murtaza-SerpentCS**

### CODE REVIEW

**Review by yvaucher — APPROVED** [review]

File: stock_picking_volume_packaging/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "summary": """
         Use volume information on potential product packaging to compute the
         volume of a stock.move""",
-    "version": "16.0.1.0.1",
+    "version": "17.0.1.0.1",
```
Comment (rousseldenis) [inline stock_picking_volume_packaging/__manifest__.py]: @Murtaza-SerpentCS Could you reset to `17.0.1.0.0`? Thanks

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**